### PR TITLE
Adaptation to model of PfemFluid

### PR DESCRIPTION
**Pull request by AFranci**
Updated for #2417 changes.

### CODE REVIEW

File: applications/PfemFluidDynamicsApplication/python_scripts/MainFluidPFEM.py [hunk 1]
```diff
@@ -384,4 +385,5 @@ def _import_project_parameters(self, input_file):
 
 
 if __name__ == "__main__":
-    Solution().Run()
+    model = Model()
```
Comment (maceligueta) [inline applications/PfemFluidDynamicsApplication/python_scripts/MainFluidPFEM.py]: I am not in the branch and I can't test it, but it looks like `Model()` wouldn't be recognized here, right?
```suggestion
    model = KratosMultiphysics.Model()
```

Comment (AFranci) [reply applications/PfemFluidDynamicsApplication/python_scripts/MainFluidPFEM.py]: Thanks Miguel Ángel. The branch was not compiling so I could not verify if our changes were working.

**Review by maceligueta — DISMISSED** [review]
We should not merge if it is not compiling.

**Comment by AFranci** [comment]
I agree, however the 'model_v3' branch was already not compilying before this merge. @RiccardoRossi will merge this pfem branch whether/when he considers it appropriate.

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**